src/encoded/static/components/filegallery.js

```diff
@@ -1384,11 +1392,29 @@ export const FileGallery = ({
         }
     }, [fileQuery, files]);
 
+    useMount(() => {
+        // Request the audits for all given analyses.
+        if (usedAnalyses && usedAnalyses.length > 0) {
```

Comment:
August will be great because then we can use Optional chaining to stop having to do this; not be stuck in having to appease IE11.